src/core/helpers/Vector2.spec.ts

```diff
@@ -44,4 +44,13 @@ describe('Vector2', () => {
       expect(result).to.deep.equal({ x: 0, y: 0 });
     });
   });
+
+  describe('equals', () => {
```

Comment:
can you throw in the negative as well? Check that it returns false when they are not matching?

Reply:
I've committed this change, thanks. :)